docs/self-managed/operational-guides/data-purge/data-purge.md

```diff
@@ -0,0 +1,189 @@
+---
+id: data-purge
+title: "Data purge"
+description: "Purge data from your cluster"
+---
+
+The data purge feature allows you to delete all runtime and historical data from your cluster. This operation resets the cluster to an empty state while maintaining the original topology.
```

Comment:
:wrench: We should emphasize the latter part - while yes it deletes data, it's more that it restores the cluster to a "fresh" state. So there will be _some_ things there after - e.g. the default user (if so configured), default tenant, some schema metadata, etc.